clients/imodels-client-management/src/base/internal/AxiosRestClient.ts

```diff
@@ -78,7 +78,7 @@ export class AxiosRestClient implements RestClient {
         const parsedError: Error = this._parseErrorFunc({ statusCode: error.response?.status, body: error.response?.data });
```

Comment:
This part seems incorrect. It can be axios error, but not have response, since it's optional (e.g., if we are getting ECONNRESET/ECONNREFUSD).

Reply:
Expanded error parsing function signature to accept original error instance
https://github.com/iTwin/imodels-clients/pull/160#discussion_r1178990374